docs/source/geo_def.rst

```diff
@@ -63,22 +62,25 @@ Creating an area definition:
 
 pyresample.utils
 ****************
-The utils module of pyresample has convenience functions for constructing
-area defintions. The function **get_area_def** can construct an area definition
-based on area extent and a proj4-string or a list of proj4 arguments.
+
+The :mod:`utils <utils>` module of Pyresample
+has convenience functions for constructing area definitions. The function
+:mod:`get_area_def <utils.get_area_def>` can construct an
+:mod:`AreaDefinition <geometry.AreaDefinition>` object based on
+area_extent and a proj4-string/dict or a list of proj4 arguments.
 
 .. doctest::
 
  >>> from pyresample import utils
  >>> area_id = 'ease_sh'
  >>> description = 'Antarctic EASE grid'
  >>> proj_id = 'ease_sh'
- >>> projection = '+proj=laea +lat_0=-90 +lon_0=0 +a=6371228.0 +units=m'
- >>> x_size = 425
- >>> y_size = 425
- >>> area_extent = (-5326849.0625,-5326849.0625,5326849.0625,5326849.0625)
- >>> area_def = utils.get_area_def(area_id, description, proj_id, projection,
- ...                  			   x_size, y_size, area_extent)
+ >>> proj_string = '+a=6371228.0 +units=m +lon_0=0 +proj=laea +lat_0=-90'
```

Comment:
Is there a reason you reordered these? The original order is fairly common and is easier to understand from a user perspective (proj first, then other parameters, then units last).

Reply:
I used what was already in the docs for ordering, but I can order it as you suggested!